server/lib/notifications/agents/pushover.ts

```diff
@@ -43,10 +48,32 @@ class PushoverAgent
     return true;
   }
 
-  private getNotificationPayload(
+  private async getImagePayload(
+    imageUrl: string
+  ): Promise<Partial<PushoverImagePayload>> {
+    try {
+      const response = await axios.get(imageUrl, {
+        responseType: 'arraybuffer',
+      });
+      const base64 = Buffer.from(response.data, 'binary').toString('base64');
+      return {
+        attachment_base64: base64,
+        attachment_type: response.headers['content-type'],
```

Comment:
```suggestion
        attachment_type: response.headers['Content-Type']?.toString(),
```
Small nitpick here, just to stay consistent with what Axios provides.

Reply:
The header was all lowercase in the few cases I tested, I have updated it to check for both formats